components/token/OnlyUserToggle.tsx

```diff
@@ -0,0 +1,32 @@
+import { Flex, Switch, Text } from 'components/primitives'
+import { FC } from 'react'
+
+type Props = {
+  checked: boolean
+  onCheckedChange: (checked: boolean) => void
+}
+
+export const OnlyUserToggle: FC<Props> = ({ checked, onCheckedChange }) => {
```

Comment:
The wording here is a bit weird. The way I see it the toggle is for displaying your own orders. Maybe it should be called OnlyUserOrdersToggle ?